### ISSUE TITLE
[fvt]2.12.2:xcatprobe could not surpport xcatmn command on ubuntu

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#1687]
env:all ubuntu environment
build:root@c910f03c17k27:/opt/xcat/probe/subcmds# lsxcatd -v
Version 2.12.2 (git commit 79426ad21e602b8a1bf5c02084ffb8c8ca9bf01e, built Thu Aug 11 06:17:21 EDT 2016)
root@c910f03c17k27:/opt/xcat/probe/subcmds# 
How to reproduce:
```
root@c910f03c17k27:/opt/xcat/probe/subcmds# xcatprobe  xcatmn -l
Unsupported sub command: xcatmn
Supported sub commands are:
discovery        Do probe for discovery process, including pre-check for required configuration and realtime monitor of
                 discovery process.Before using this command, please install nslookup command ahead. Currently, this
                 command does not support hierarchy.
detect_dhcpd     detect_dhcpd can be used to detect the dhcp server in a network for a specific mac address. Before
                 using this command, please install tcpdump command ahead. The operating system supported are redhat,
                 sles, ubuntu and debian.
switch-macmap    To retrieve MAC address mapping for the specified switch, or all the switches defined in switches table
                 in xCAT db. Currently, this command does not support hierarchy.
osdeploy         Do probe for os provision process, realtime monitor of os provision process.
image            Use this command to check if specified compute nodes have the same images installed or if compute nodes
                 are installed with the same image as defined on MN.
root@c910f03c17k27:/opt/xcat/probe/subcmds# lsxcatd -v
Version 2.12.2 (git commit 79426ad21e602b8a1bf5c02084ffb8c8ca9bf01e, built Thu Aug 11 06:17:21 EDT 2016)
---------------------------> could not use xcatmn command on xcatprobe framework

```

**Comment by junxiawang** [comment]
root@c910f03c17k27:/opt/xcat/probe/subcmds# /opt/xcat/probe/subcmds/xcatmn -i eth0
Can't locate xCAT/ServiceNodeUtils.pm in @INC (you may need to install the xCAT::ServiceNodeUtils module) (@INC contains: /opt/xcat/probe/lib/perl /etc/perl /usr/local/lib/powerpc64le-linux-gnu/perl/5.22.1 /usr/local/share/perl/5.22.1 /usr/lib/powerpc64le-linux-gnu/perl5/5.22 /usr/share/perl5 /usr/lib/powerpc64le-linux-gnu/perl/5.22 /usr/share/perl/5.22 /usr/local/lib/site_perl /usr/lib/powerpc64le-linux-gnu/perl-base .) at /opt/xcat/probe/subcmds/xcatmn line 8.
BEGIN failed--compilation aborted at /opt/xcat/probe/subcmds/xcatmn line 8.
root@c910f03c17k27:/opt/xcat/probe/subcmds#

**Comment by hu-weihua** [comment]
Fixed by pull request #1691 . @junxiawang Could you help to verify it ? thanks

**Comment by junxiawang** [comment]
verified
